wsnet2-unity/Assets/WSNet2/Scripts/Core/Room.cs

```diff
@@ -1243,6 +1250,17 @@ private void OnEvResponse(EvResponse ev)
             });
         }
 
+        internal void ConnectionStateChanged(bool connected)
+        {
+            callbackPool.Add(() =>
+            {
+                if (OnConnectionStateChanged != null)
+                {
+                    OnConnectionStateChanged(connected);
+                }
```

Comment:
null条件演算子使っても良さそう
.NET5バージョン推奨ならC#6.0はサポートしてそうなので
```suggestion
                OnConnectionStateChanged?.Invoke();
```

Reply:
たしかに、.NETだけじゃなくUnityも現行バージョンではC#6.0以上になってますよね。
ほかの場所がifでチェックの形なので、いったん合わせてこの形にしておいて、別のPRでまとめて書き換えます。